src/features/form/validations.jsx

```diff
@@ -35,6 +35,19 @@ const isRequired = value => value ? undefined : <LocalizedMessage
   id="reactcomponents.value.required"
   defaultMessage="Required" />;
 
+const allowDecimals = value => value => {
+  let parsedValue = value ? value.toString().trim() : value;
+  // if the value is empty OR
+  // if the value contains any non-whitespace(\S) characters
+  // and if no non-Digits(\D) characters are present
+  // and if the value is a whole number
+  if ( !parsedValue || (/\S/.test(parsedValue) && !(/\D/.test(parsedValue)) && parseInt(parsedValue,10) && ((parsedValue - Math.floor(parsedValue)) === 0))) {
```

Comment:
What is the parseInt(parseValue,10) testing?  just that it can be successfully parsed as an int?  What if the parsedValue is 0?

This is also complex enough we should create a validations.test.jsx and add some tests for this.